### PR TITLE
finalizing ai-chatbot-assistant readme and updating project structure

### PR DESCRIPTION
**Pull request by vtwoptwo**

### CODE REVIEW

**Comment by vtwoptwo** [comment]
Updated Makefile, Readme (added the video and in depth instructions), added issues for troubleshooting, a contributions sections, license and small restructuring of the files :)

File: ai-chatbot-assistant/Home.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     st.write(' ')
 
 with col2:
-    st.image("./assets/jarvis.png", width=300)
+    st.image("https://raw.githubusercontent.com/IERoboticsClub/workshops/main/ai-chatbot-assistant/assets/jarvis.png", width=300)
```
Comment (dsanmart) [inline ai-chatbot-assistant/Home.py]: Fix to local path: ai-chatbot-assistant/assets/jarvis.png

File: ai-chatbot-assistant/requirements.txt
Comment (dsanmart) [inline ai-chatbot-assistant/requirements.txt:1]: Missing version of the dependencies

File: ai-chatbot-assistant/utils/jarvis_helpers.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                        Text_to_speech(plot)
+                    else:
+                        print("Failed to retrieve plot. Status code:", response.status_code)
+            
+            break
+    
+    # return a combination of all the prompts and response
+    return command, response
+
+if __name__ == '__main__':
```
Comment (dsanmart) [inline ai-chatbot-assistant/utils/jarvis_helpers.py]: I would comment this out

File: ai-chatbot-assistant/Home.py (hunk 1 above)
Comment (vtwoptwo) [inline ai-chatbot-assistant/Home.py]:
```python
 st.image("ai-chatbot-assistant/assets/jarvis.png", width=300)
```

File: ai-chatbot-assistant/utils/redis_helpers.py [hunk 3]
```diff
@@ -18,6 +19,13 @@ def connect_redis():
 
 
 def upload_to_redis(id_sentence: str, sourcepage: str, sourcefile: str, text: str, embedded_text: list, redis_conn: redis.Redis):
+    """Upload a document to Redis."""
+    # get path and check if tempdir is thre 
+    # if not, create it
```
Comment (dsanmart) [inline ai-chatbot-assistant/utils/redis_helpers.py]: Nit: Remove spaces and comments

**Review by dsanmart — CHANGES_REQUESTED** [review]
Readme looking great!! Added some minor changes comments

**Comment by vtwoptwo** [comment]
@dsanmart changes implemented

**Review by dsanmart — APPROVED** [review]

**Pull request merged**